Framework/Geometry/test/ComponentInfoBankHelpersTest.h

```diff
@@ -175,6 +175,46 @@ class ComponentInfoBankHelpersTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS(offsetFromAncestor(compInfo, ancestorIndex, currentIndex),
                      std::invalid_argument &);
   }
+
+  void test_isAncestorOf_finds_ancestor() {
+
+    auto instrument = ComponentCreationHelper::createMinimalInstrument(
+        V3D(0, 0, -10), V3D(0, 0, 0), V3D(0, 0, 10));
+    auto wrappers = InstrumentVisitor::makeWrappers(*instrument);
+
+    // instrument cache to be used with call offsetFromAncestor
+    const auto &compInfo = (*wrappers.first);
+
+    // the source is the child of the root
+    TS_ASSERT(isAncestorOf(compInfo, compInfo.root(), compInfo.source()));
+  }
+
+  void
+  test_isAncestorOf_returns_false_with_child_sibling_or_relative_proposed_as_ancestor() {
```

Comment:
you should have a comment somewhere about the fact that the root is not counted! in fact would definitely want that stated by the doxygen for `isAncestorOf`

Reply:
done